specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/iotdps.json

```diff
@@ -0,0 +1,2622 @@
+        "description": "Deletes the Provisioning Service.",
+        "x-ms-long-running-operation": true,
+        "operationId": "IotDpsResource_Delete",
+        "x-ms-examples": {
+          "DPSDelete": {
+            "$ref": "./examples/DPSDelete.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "provisioningServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of provisioning service to delete."
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Resource group identifier."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The provisioning service resource provider always returns a 202 Accepted status code with valid Location and Retry-After headers. The resource provider also sets the Azure-AsyncOperation header with a URL that points to the operation resource for this operation. Subsequent GET attempts on the resource after a DELETE operation return a resource representation that indicates a transitional provisioning state (such as Terminating). To retrieve the status of the operation, a client can either poll the URL returned in the Location header after the Retry-After interval, get the provisioning service status directly, or query the operation resource."
+          },
+          "200": {
+            "description": "This is returned as a response to the status polling request for the delete operation. The body contains the resource representation that indicates a transitional provisioning state."
+          },
+          "204": {
+            "description": "Once the long running delete operation completes successfully, a 204 No Content status code is returned when the status polling request finds the provisioning service metadata in the service and the status of the delete operation is set to a completed state."
+          },
+          "404": {
+            "description": "After the long running delete operation completes successfully, a 404 Not Found is returned when the status polling request no longer finds the provisioning service metadata in the service."
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/certificates": {
```

Comment:
Please add x-ms-pageable to all List apis even if paging is not supported.
https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html?q=x-ms-pageable#oapi009-always-add-x-ms-pageable-to-list-calls

Reply:
Temporarily resolved. See details in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/12378#discussion_r560402312)